src/Microsoft.AspNetCore.JsonPatch/JsonPatchDocument.cs

```diff
@@ -24,26 +24,25 @@ public class JsonPatchDocument : IJsonPatchDocument
         [JsonIgnore]
         public IContractResolver ContractResolver { get; set; }
 
-        public JsonPatchDocument()
+        /// <summary>
+        /// Gets or sets the AdapterFactory to use when processing this document
+        /// </summary>
+        [JsonIgnore]
+        public IAdapterFactory AdapterFactory { get; set; }
+
+        public JsonPatchDocument():this(new List<Operation>(), new DefaultContractResolver())
```

Comment:
Formatting, please put the `: this`... and everything after on a separate line (3-4 cases of this in the PR)

Reply:
Please exclude changes to this file, for the reasons I mentioned earlier.